### PR TITLE
Support STORE and EXECUTE commands

### PR DESCRIPTION
**Pull request by sazzad16**

### CODE REVIEW

File: src/main/java/com/redislabs/redisai/Model.java [hunk 1]
```diff
@@ -234,6 +234,48 @@ public void setMinBatchSize(long minbatchsize) {
...
+    if (tag != null) {
+      args.add(Keyword.TAG.getRaw());
+      args.add(SafeEncoder.encode(tag));
+    }
+    if (batchSize > 0) {
+      args.add(Keyword.BATCHSIZE.getRaw());
+      args.add(Protocol.toByteArray(batchSize));
+      if (minBatchSize > 0) {
+        args.add(Keyword.MINBATCHSIZE.getRaw());
+        args.add(Protocol.toByteArray(minBatchSize));
```
Comment (DvirDukhan) [inline src/main/java/com/redislabs/redisai/Model.java:257]: missing `MINBATCHTIMEOUT` https://oss.redislabs.com/redisai/commands/#aimodelstore

File: src/main/java/com/redislabs/redisai/Model.java [hunk 2]
```diff
@@ -234,6 +234,48 @@ public void setMinBatchSize(long minbatchsize) {
...
+    }
+    if (outputs != null && outputs.length > 0) {
+      args.add(Keyword.OUTPUTS.getRaw());
+      args.add(Protocol.toByteArray(outputs.length));
+      for (String output : outputs) {
+        args.add(SafeEncoder.encode(output));
+      }
+    }
+    args.add(Keyword.BLOB.getRaw());
+    args.add(blob);
```
Comment (DvirDukhan) [inline src/main/java/com/redislabs/redisai/Model.java:275]: blob is `byte[]`, where is the split to chuncks?

Comment (sazzad16) [reply src/main/java/com/redislabs/redisai/Model.java:318]: @DvirDukhan My preference is to do that in separate PR.

File: src/test/java/com/redislabs/redisai/DagTestV2.java [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+    Tensor tB = new Tensor(DataType.FLOAT, new long[] {2, 3}, new float[][] {{1, 2, 3}, {4, 5, 6}});
+    dag.setTensor(keyA, tA);
+    dag.setTensor(keyB, tB);
+    dag.executeModel("mul", new String[] {keyA, keyB}, new String[] {keyC});
+    //    dag.getTensor(keyC);
+    List<?> result = client.dagExecuteReadOnly(null, new String[] {"mul"}, dag);
+    Assert.assertArrayEquals("OK".getBytes(), (byte[]) result.get(0));
+    Assert.assertArrayEquals("OK".getBytes(), (byte[]) result.get(1));
+    Assert.assertEquals(RedisAIException.class, result.get(2).getClass());
+    //    Assert.assertArrayEquals("NA".getBytes(), (byte[]) result.get(3));
```
Comment (DvirDukhan) [inline src/test/java/com/redislabs/redisai/DagTestV2.java:166]: for the time being, what will `result.get(3)` will return?

**Review by DvirDukhan — APPROVED** [review]

File: src/test/java/com/redislabs/redisai/DagTestV2.java (hunk 3 above)
Comment (sazzad16) [reply src/test/java/com/redislabs/redisai/DagTestV2.java:166]: `NA` as a byte array.

Comment (sazzad16) [reply src/test/java/com/redislabs/redisai/DagTestV2.java:166]: @DvirDukhan `NA` as a byte array. Same as OK, two and three lines above.

**Pull request merged**